scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/ScaladocParser.scala

```diff
@@ -57,6 +60,10 @@ object ScaladocParser {
     P((!codeLineEnd ~ AnyChar).rep.!)
   }
 
+  private def markdownCodeLineParser[_: P]: P[String] = {
+    P(!markdownCodeSuffix ~ (!nl ~ AnyChar).rep.!)
+  }
+
   private def codeExprParser[_: P]: P[CodeExpr] = {
     def pattern = codePrefix ~ hspaces0 ~ codeLineParser ~ codeSuffix ~ punctParser.!
```

Comment:
We should also add support for parsing expression witihin `` as suggested by @kitbellew

Reply:
I think in that case we also need to distinguish between ` `` ` and ` {{{}}} `. Should I create a new element for markdown code expression or add the information about the symbol to the existing `CodeExpr` class?